### ISSUE TITLE
adapter Select to content provider query.

### ISSUE THREAD

**Issue by jiqimaogou** [Raizlabs/DBFlow#119]
```
List<DeviceObject> devices = new Select().from(DeviceObject.class)
  .where(
      Condition.column(DeviceObject$Table.NAME).eq("Samsung-Galaxy-S5"), 
      Condition.column(DeviceObject$Table.CARRIER).eq("T-Mobile")).queryList();
```

can you add Select these methods:
1. Uri getContentUri()
2. String[] getProjection()
3. String getSelection()
4. String[] getSelectionArgs()
5. String getSortOrder()

to get data from contentprovider. or let select class support for contentprovider.

why contentprovider? because I sometime use cursorloader which is only support for ContentProvider.

**Comment by agrosner** [comment]
I am sorry. I don't understand your question. Can you rephrase.

**Comment by jiqimaogou** [comment]
sorry for my english,
https://github.com/foxykeep/DataDroid/blob/master/DataDroidPoC/src/com/foxykeep/datadroidpoc/data/provider/util/ProviderCriteria.java
I say I think it can extend the select like ProviderCriteria, so I can use select class to query contentprovider. the select class only support for rawquery, but not sqldatabase.query.

**Comment by agrosner** [comment]
you mean ```Condition```? (https://github.com/Raizlabs/DBFlow/blob/master/usage/Conditions.md)

**Comment by agrosner** [comment]
and yes there is for conditions in a ```@ContentProvider``` using ````@PathSegment``` (https://github.com/Raizlabs/DBFlow/blob/master/usage/ContentProviderGenerators.md)

**Comment by agrosner** [comment]
closing issue.
